docs/release-notes.md

```diff
@@ -7,15 +7,12 @@ This section outlines key features changes between versions 1.3 and 1.4 for Open
 
 ### Additions
 #### RPS & MPS
-- **Containerized Scaling Support:**  In 1.2 we released a preview version of scaling, a first look at supporting large-scale deployments. In 1.4 we've updated scaling capabilities with support for Docker Swarm and Kubernetes deployments.
-With this feature, scale the MPS and RPS microservices as needed to support an entire fleet of Intel® AMT devices.
-
-The toolkit enables robust scaling through:
-    * A new component (MPS Router) that acts as a reverse proxy between the API Gateway and the MPS. More on MPS Router below.
-    * Support for deploying cloud based components and dependencies to Docker Swarm and Kubernetes. Components include MPS, RPS, MPS Router, API Gateway, Vault, PostgreSQL.  
-    * Cloud provider agnostic [deployment documentation](https://open-amt-cloud-toolkit.github.io/docs/1.4/Kubernetes/kubernetes/).
-    * Improved routing of KVM and SOL sessions when multiple instances of MPS are deployed.
-    * An increase in the number of CIRA connections that can be handled per MPS instance. Detailed guidance will be provided with the LTS release.
+- **Containerized Scaling Support:**  In 1.2 we released a preview version of scaling, a first look at supporting large-scale deployments. In 1.4 we've updated scaling capabilities with support for Docker Swarm and Kubernetes deployments.  With this feature, scale the MPS and RPS microservices as needed to support an entire fleet of Intel® AMT devices.  The toolkit enables robust scaling through:
+   *  A new component (MPS Router) that acts as a reverse proxy between the API Gateway and the MPS. More on MPS Router below.
+   *  Support for deploying cloud based components and dependencies to Docker Swarm and Kubernetes. Components include MPS, RPS, MPS Router, API Gateway, Vault, PostgreSQL.  
+   *  Cloud provider agnostic [deployment documentation](https://open-amt-cloud-toolkit.github.io/docs/1.4/Kubernetes/kubernetes/).
+   *  Improved routing of KVM and SOL sessions when multiple instances of MPS are deployed.
+   *  An increase in the number of CIRA connections that can be handled per MPS instance. Detailed guidance will be provided with the LTS release.
 
 #### MPS Router
 - **Reverse Proxy:** In this release we are adding a new required component to the toolkit deployment, the MPS Router.  This small containerized GO serivce acts as a reverse proxy between the API Gateway and MPS.  When Intel&reg; AMT devices make a CIRA connection with MPS, the MPS registers the device GUID and its MPS instance ID with the MPS Router.  All MPS API calls coming from the API Gateway are first sent to the MPS Router before being forwarded to the correct MPS instance.  As with all components of the Open AMT Cloud Toolkit, this service is open source and is located in the [MPS Router](https://github.com/open-amt-cloud-toolkit/mps-router) repository.
```

Comment:
This small containerized GO serivce acts ->  This small containerized Golang-based service acts

Reply:
Done